pkg/cmd/server/start/start_master.go

```diff
@@ -446,6 +446,7 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 
 	// Must start policy caching immediately
 	oc.Informers.StartCore(utilwait.NeverStop)
+	oc.Informers.Start(utilwait.NeverStop)
```

Comment:
This should have happened here: https://github.com/openshift/origin/blob/master/pkg/cmd/server/start/start_master.go#L418 .  Is there some sequence requirement that I'm not seeing or was it just in a weird spot?

That location is definitely non-obvious, but we can only start these *after* the controllers have been initialized and the controllers are optional when starting the API, so it starts to look a little ugly.